### ISSUE TITLE
A signed in user should have a default Library folder

### ISSUE THREAD

**Issue by elsigh** [elsigh/chartcomposer#108]
We can create it in their dropbox `/Apps/ChartComposer/.preferences` file so it's in a consistent place.

It should always exist, and if the user deletes it, we'll recreate it upon sign in.

**Comment by davesyoung** [comment]
We should be able to respect them the deleting songs from it. Just in case they don't like the default song list that is available to them and want to remove a Christian or child song.

**Comment by elsigh** [comment]
yeah they should be able to delete songs (which you cannot do now fwiw).
